boards/arc/arduino_101_sss/arduino_101_sss.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: arduino_101_sss
+name: arduino_101_sss
```

Comment:
if identifier and name are always the same, loose one.
If it is always goign to look like the file name, default to the file name instead of having yet another field.